pkg/ctl/create/cluster.go

```diff
@@ -49,45 +43,22 @@ func createClusterCmd() *cobra.Command {
 	fs := cmd.Flags()
 
 	exampleClusterName := utils.ClusterName("", "")
-
 	fs.StringVarP(&cfg.ClusterName, "name", "n", "", fmt.Sprintf("EKS cluster name (generated if unspecified, e.g. %q)", exampleClusterName))
 
-	fs.StringVarP(&cfg.Region, "region", "r", "", "AWS region")
-	fs.StringVarP(&cfg.Profile, "profile", "p", "", "AWS credentials profile to use (overrides the AWS_PROFILE environment variable)")
-	fs.StringToStringVarP(&cfg.Tags, "tags", "", map[string]string{}, `A list of KV pairs used to tag the AWS resources (e.g. "Owner=John Doe,Team=Some Team")`)
-
-	fs.StringVarP(&ng.InstanceType, "node-type", "t", defaultNodeType, "node instance type")
-	fs.IntVarP(&ng.DesiredCapacity, "nodes", "N", api.DefaultNodeCount, "total number of nodes (for a static ASG)")
-
-	// TODO: https://github.com/weaveworks/eksctl/issues/28
-	fs.IntVarP(&ng.MinSize, "nodes-min", "m", 0, "minimum nodes in ASG")
-	fs.IntVarP(&ng.MaxSize, "nodes-max", "M", 0, "maximum nodes in ASG")
-
-	fs.IntVarP(&ng.VolumeSize, "node-volume-size", "", 0, "Node volume size (in GB)")
-	fs.IntVar(&ng.MaxPodsPerNode, "max-pods-per-node", 0, "maximum number of pods per node (set automatically if unspecified)")
-	fs.StringSliceVar(&availabilityZones, "zones", nil, "(auto-select if unspecified)")
-
-	fs.BoolVar(&ng.AllowSSH, "ssh-access", false, "control SSH access for nodes")
-	fs.StringVar(&ng.SSHPublicKeyPath, "ssh-public-key", defaultSSHPublicKey, "SSH public key to use for nodes (import from local path, or use existing EC2 key pair)")
+	addCommonCreateFlags(fs, cfg, ng)
```

Comment:
We might benefit from a separate PR that does some refactoring and cleanup to fix #298. I think that `--region`, `--profile` are somewhat common globally, only a few commands don't inherit those.